### ISSUE TITLE
OCI image compatibility

### ISSUE THREAD

**Issue by romdum** [argoproj-labs/argocd-image-updater#236]
**Is your feature request related to a problem? Please describe.**

We use OCI image, when ArgoCD Image updater make a request to our image registry, it respond this error : 
```
Error fetching metadata for private/our-project-image:our-tag - neither V1 or V2 manifest returned by registry: Get "https://registry.url/v2/private/our-project-image/manifests/our-tag": http: non-successful response (status=404 body="{"errors":[{"code":"MANIFEST_UNKNOWN","message":"OCI manifest found, but accept header does not support OCI manifests"}]}")"  
```

I reproduced the problem by calling the registry API without the header `Accept: 
application/vnd.oci.image.manifest.v1+json`. If I add the header above, the registry return the manifest. 

**Describe the solution you'd like**
When ArgoCD Image Update make a request to the registry, it should add the header `Accept: 
application/vnd.oci.image.manifest.v1+json` to be able to get OCI image manifest.

**Describe alternatives you've considered**
For now, the any option we have is to build docker image instead of OCI image.

**Additional context**
Registry : Harbor V2
ArgoCD Image Updater Version : 0.9.4
ArgoCD Version : 2.0.5

**Comment by jannfis** [comment]
Yeah, I do agree that we need to support OCI. Thanks also for doing some research on that.

**Comment by jannfis** [comment]
This has been implemented by #249 and will be part of v0.11.0 release.

**Comment by jannfis** [comment]
v0.11.0 has been released, closing.
